### PR TITLE
add todos enzyme tests

### PR DESCRIPTION
**Pull request by mjw56**
This PR adds `enzyme` tests for the `todos` example `components`

This is part of #1481

### CODE REVIEW

**Comment by gaearon** [comment]
I specifically didn’t mention `todos` in #1481 because @jontewks was already working on it in #1471. I should have communicated this better :disappointed_relieved: .

Ultimately we want to use enzyme in all our tests so this probably supersedes @jontewks’s work. I apologize for not handling it better.

Can you please make sure that this includes whatever tests existed in #1471?

**Comment by mjw56** [comment]
Yeah this was my fault, I should have checked the open PR's first. I reviewed #1471 and it looks like this includes all the tests from there.

**Comment by whoisandie** [comment]
Aren't the component tests for todo-with-undo the same as todos? Trying to figure out if there's any other component test that has to be added. The action and reducer tests are missing though. Since redux-undo is already tested, minor tweaks in action and reducer tests might be sufficient.

File: examples/todos/package.json [hunk 1]
```diff
@@ -31,9 +31,10 @@
     "babel-preset-react-hmre": "^1.1.1",
     "babel-register": "^6.3.13",
     "cross-env": "^1.0.7",
+    "enzyme": "^2.0.0",
     "expect": "^1.8.0",
     "express": "^4.13.3",
-    "jsdom": "^5.6.1",
+    "jsdom": "^8.1.0",
```
Comment (gaearon) [inline examples/todos/package.json]: Shouldn’t we be able to get rid of `jsdom` now?

Comment (mjw56) [inline examples/todos/package.json]: Yeah, we can safely remove ```jsdom``` now. Good catch, thanks. I've updated the PR with 0baa696 to reflect this.

**Comment by timdorr** [comment]
Can this be reworked to also remove/replace [the existing React Test Utils tests](https://github.com/reactjs/redux/tree/master/examples/todomvc/test/components)?

**Comment by mjw56** [comment]
@timdorr sure, we can do that. If anyone else is interested, please feel free to chime in. I wouldn't be able to start these until tomorrow.

**Comment by mjw56** [comment]
@timdorr I started by converting one file, does this look ok to you?

**Comment by timdorr** [comment]
Thanks for the efforts @mjw56. We're going to go down the route of using react-test-renderer instead.

**Pull request closed without merging**